circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -55,6 +55,49 @@
 
   private final static Logger LOG = LoggerFactory.getLogger(S3S3Copier.class);
 
+  private static class BytesTransferStateChangeListener implements TransferStateChangeListener {
+
+    private final S3ObjectSummary s3ObjectSummary;
+    private final AmazonS3URI targetS3Uri;
+    private final String targetKey;
+
+    public BytesTransferStateChangeListener(
```

Comment:
Does this need to be public? Could we make it private or default?

Reply:
I made it private